src/main/java/appeng/core/api/definitions/ApiItems.java

```diff
@@ -133,6 +134,7 @@ public ApiItems( final DefinitionConstructor constructor )
 
 		this.entropyManipulator = constructor.registerItemDefinition( new ToolEntropyManipulator() );
 		this.wirelessTerminal = constructor.registerItemDefinition( new ToolWirelessTerminal() );
+		this.coloredWirelessTerminal = constructor.constructColoredDefinition(this.wirelessTerminal, 0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ')' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '(' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)